### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by programad**
Documentation review. A few points to consider:

- Customizar -> Personalizar?
- Utilitários -> Classes?
- Absolute links vs vscode lint
- Links from usage to getting-started are relative to the root
- Some titles have hidden links. Is this intentional?
- Duplicated cursor section on Styling page. Should it be filters?

Also, the menu bug was fixed on this branch.

### CODE REVIEW

File: packages/ignition/src/app/components/documentation/sidebar-left.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export const SidebarLeft = ({ lang }: PropsWithLang) => {
                 : docs.filter(
                     (doc: any) =>
                       doc[1][lang].group === mainDocEnSlug &&
-                      doc[1][lang].group != doc[1][lang].slug
+                      doc[1][lang].group != doc[1][lang].enslug
```
Comment (amorimjj) [inline packages/ignition/src/app/components/documentation/sidebar-left.tsx:90]: I'm going to believe on you on this one. No clue

File: packages/ignition/src/app/locales/docs/usage/components/responsive-design-explanation.pt-br.mdx [hunk 2]
```diff
@@ -7,6 +7,6 @@ order: 6
 activeSelector: group-has-[.docs-responsive-design]:active-content
 ---
 
-O exemplo acima funciona usando breakpoints do Tailwind, incluindo um com valor arbitrário, customizando o desing para dirente tamanhos de telas.
+O exemplo acima funciona usando breakpoints do Tailwind, mesmo que seja arbitrário, customizando o desing para diferentes tamanhos de telas.
```
Comment (amorimjj) [inline packages/ignition/src/app/locales/docs/usage/components/responsive-design-explanation.pt-br.mdx:10]: Acho que a gente precisa melhorar isso... O exemplo inclui breakpoints que são padrões no Tailwind e um configurado arbitrariamente usando `md-[valor-arbitrario]:...`. O texto estava ruim, mas ainda ficou confuso.

File: packages/ignition/src/app/locales/docs/usage/components/styling-filters.pt-br.mdx [hunk 3]
```diff
@@ -8,6 +8,6 @@ activeSelector: group-has-[.docs-styling]:active-content
 ---
 
 #### [Filtros](https://tailwindcss.com/docs/blur)
-Filtros e ícones
+Filtros e ícones.
 
-*NativeWind doesn't support filters, therefore this functionality isn't available for React Native."*
+*NativeWind doesn't support filters, logo essa funcionalidade não está disponível para React Native*
```
Comment (amorimjj) [inline packages/ignition/src/app/locales/docs/usage/components/styling-filters.pt-br.mdx:13]:
```suggestion
*NativeWind não suporta filtros, logo essa funcionalidade não está disponível para React Native*
```

**Review by amorimjj — CHANGES_REQUESTED** [review]
I've drop few comments for you

File: packages/ignition/src/app/locales/docs/usage/usage.pt-br.mdx [hunk 4]
```diff
@@ -18,15 +18,16 @@ O <RocketIconsText /> é uma biblioteca de ícones como parte de uma solução q
 rodem tanto na Web quanto no Mobile, sem alterações, economizando recursos e tempo de desenvolvimento, possibilitando que
 o foco seja direcionado ao que importa. A sua aplicação!
 <br />
-Como desenvolvedores fãs do Tailwind, foi uma processo natural usar esse poderoso framework para entregar ao ícones um grande
+Como desenvolvedores fãs do Tailwind, foi um processo natural usar esse poderoso framework para dar aos ícones um grande
 poder de personalização e customização.
 
-> Lembro de ter ficado maravilhado quando entendi o funcionando do Tailwind e apesar de não ter nenhuma ligação com a empresa
+> Lembro de ter ficado maravilhado quando entendemos o funcionando do Tailwind e apesar de não ter nenhuma ligação com a empresa
```
[the remark this answers is not in the document]
Comment (programad) [reply packages/ignition/src/app/locales/docs/usage/usage.pt-br.mdx:24]: Eu havia alterado pro plural, porém percebi logo em seguida que era um quote e voltei. Isso ficou sem querer, Vou corrigir.

File: packages/ignition/src/app/components/documentation/sidebar-left.tsx (hunk 1 above)
Comment (programad) [reply packages/ignition/src/app/components/documentation/sidebar-left.tsx:90]: This was causing the bug of the lone menu being rendered at the end of Usage. It was looking for a parent "usando" which was never found in pt-br.

**Review by amorimjj — APPROVED** [review]
After our chat, I'm ok.

Code 👍

**Pull request merged**